src/ott/geometry/costs.py

```diff
@@ -502,12 +507,23 @@ def f_h(
 
     return f_h
 
+  @property
+  def lam(self) -> float:
+    """Strength of the regularization."""
```

Comment:
Because this is not necessarily super intuitive, it might be worth adding some basic explanation here that `lam` is converted to `alpha` in `__init__`, and stored there subsequently?

Reply:
Done!